home/utils.py

```diff
@@ -2,25 +2,36 @@
 import csv
 import io
 from dataclasses import dataclass
+from io import BytesIO
 from math import ceil
 from typing import List, Tuple, Union
 
 from django.http import HttpResponse
+from openpyxl import load_workbook
 from openpyxl.styles import Border, Color, Font, NamedStyle, PatternFill, Side
 from openpyxl.utils import get_column_letter
 from openpyxl.workbook import Workbook
 from openpyxl.worksheet.worksheet import Worksheet
 from taggit.models import Tag
 from wagtail import blocks
 from wagtail.documents.models import Document
-from wagtail.images.blocks import ImageChooserBlock
+from wagtail.documents.blocks import DocumentChooserBlock
 from wagtail.images.models import Image
 from wagtail.models import Locale
 from wagtail.query import PageQuerySet
 from wagtail.rich_text import RichText
 from wagtailmedia.models import Media
+from wagtail.images.blocks import ImageChooserBlock
+from home.models import (  # isort:skip
+    ContentPage,
+    ContentPageIndex,
+    ContentQuickReply,
+    ContentTrigger,
+    HomePage,
+    MediaBlock,
+    Page,
+)
```

Comment:
Have a look at https://github.com/praekeltfoundation/vumi2/blob/main/pyproject.toml#L48-L50 if you want isort and black to work together